### PR TITLE
Add smarty blocks to document template

### PR DESCRIPTION
**Pull request by larsbo**
### 1. Why is this change necessary?
More precise expansion of these document parts.

### 2. What does this change do, exactly?
Adds two new block around the document header logo and the document header.

### 3. Describe each step to reproduce the issue or behaviour.
Try to extend the now wrapped document parts.

### 6. Checklist

- [ ] I have written tests and verified that they fail without my change
- [x] I have squashed any insignificant commits
- [x] This change has comments for package types, values, functions, and non-obvious lines of code
- [x] I have read the contribution requirements and fulfil them.

### CODE REVIEW

File: themes/Frontend/Bare/documents/index.tpl [hunk 1]
```diff
@@ -82,9 +82,12 @@ td.head  {
     {* @Deprecated: Wrong variable will be removed in next major release *}
     {$postions = $positions}
 
+    {block name="document_index_head_logo"}
     <div id="head_logo">
```
Comment (shyim) [inline themes/Frontend/Bare/documents/index.tpl:86]: Indention

File: themes/Frontend/Bare/documents/index.tpl [hunk 2]
```diff
@@ -82,9 +82,12 @@ td.head  {
...
         {$Containers.Logo.value}
     </div>
+    {/block}
+    {block name="document_index_head_wrapper"}
     <div id="header">
```
Comment (shyim) [inline themes/Frontend/Bare/documents/index.tpl:91]: Indention

File: themes/Frontend/Bare/documents/index.tpl (hunk 1 above)
Comment (shyim) [reply themes/Frontend/Bare/documents/index.tpl:86]: I fix it internally :sweat_smile:

**Comment by shyim** [comment]
Thanks for your Contribution! :blue_heart:

File: themes/Frontend/Bare/documents/index.tpl (hunk 1 above)
Comment (larsbo) [reply themes/Frontend/Bare/documents/index.tpl:86]: I saw many PRs without updating the indentation. Next time I'll indent the code 👍

**Pull request closed without merging**